src/index.js

```diff
@@ -63,31 +63,28 @@ export function app(state, actions, view, container) {
   }
 
   function repaint() {
-    if (view && !lock) {
```

Comment:
Is view a required parameter now? This can lead to `TypeError: view is not a function` later when using Hyperapp without a view. 😕

Reply:
@okwolf Yes, it's required now! So, don't use hyperapp without a view.

Btw, you don't need to think about this function as solely serving the purpose of a view. You can think of it as an event that is subscribing to state changes.

Reply:
@JorgeBucaran thanks for clearing that up! Is there a reason we are [checking if `container` is defined](https://github.com/hyperapp/hyperapp/blob/c7741845556c3f101e931ed7e29ab183c56317ff/src/index.js#L58) then? I think we might want to skip calling the `view` function too and not just `patch`? 🤔 

Either way, should we make this clearer in the docs regarding what's required?

Reply:
Ping @Swizz

Reply:
@okwolf The idea is to avoid patching `null` or `undefined` if no container is given. But when would you do that? I think @Swizz could answer that.

Reply:
We made this choice to allow users to use hyperapp as a State manager without the Vdom engine. 
Basically you can now use the view hook to call you own render engine (canvas, native, worker events, etc..).

We removed the optional container capabilities because of if you attempt to use a HTML element that doesn't exist as container the whole body will be used as container and this can lead to an unexpected behavior. 

Finally, we can said you would use hyperapp without a container but why would you use hyperapp without a view/render ?